check_cpu_percentage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
I would suggest to keep it as `python` only for backward compatibility for using it with python2.7.

Reason: Though 2.7 is eol by Jan 2020, not everyone will adopt migrate to 3 pretty soon.

Reply:
ok, of course, you are right.  I put the shabang automatically.